### DISCUSSION TITLE
OpenAI與Bing合作開發有對話能力的搜尋引擎

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#1084] [科技新知]
新的Bing版本將融合ChatGPT，與我們現今所使用羅列連結與簡短節錄文字的搜尋引擎不同，ChatGPT可以直接回答搜尋查詢。
之前老師上課一直提到的ChatGPT又要有新的應用了，文章也有提到ChatGPT原本所與人的對話資訊都不一定是對的，只能參考，因為他只是一個大型的語言訓練模型。
我自己是覺得我如果要搜尋資料可能還是比較偏好原本的網頁條列式的搜尋方式，因為感覺能確認資料來源以及判讀正確性與否。雖然需要花費較多的時間，但至少親自確認過的答案我比較願意相信。
快來留言你們會不會想使用對話型的搜尋方式吧！

https://www.bnext.com.tw/article/73648/micro-ai-google-20230107

**Comment by Jzd314** [comment]
我也是比較偏向原本的網頁條列式搜尋方式，可以找到比較多元的資料及看法

**Comment by harrylee1971** [comment]
有新型的搜尋方式肯定是不錯的，可以找到各種方式去促進社會更為進步
